examples/proc_raw.py

```diff
@@ -22,6 +26,9 @@
 import sys
 import os
 from itertools import cycle
+import pyspecdata as psd
+
+psd.init_logging(level="debug")
```

Comment:
is this necessary for the script to run?

Reply:
it's needed to understand what is going on.